### DISCUSSION TITLE
How to get stared with Jaeger

### DISCUSSION THREAD

**Discussion by Priyansurout** [cncf/mentoring#1057] [Q&A] [answered]
Hello 

I am Priyansu Rout, a first-year computer science student.

I am going through CNCF project ideas and have found Jaeger "Implement ClickHouse Support" idea. I did love to work on this project! Can you please suggest how to proceed?

Thank you

**Comment by yurishkuro** [comment, marked as answer]
discussed on Slack https://cloud-native.slack.com/archives/CGG7NFUJ3/p1690558506637679

**Reply by nate-double-u** [reply]
@Priyansurout, I'd also recommend reading the _[Do you want to start contributing to open source and need help figuring out where to begin?](https://contribute.cncf.io/contributors/getting-started/)_ article, it's not Jaeger specific, but gives a good overview for getting started in open source.
